recipes/mariadb-connector-c/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mariadb-connector-c" %}
+{% set version = "3.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mariadb-corporation/mariadb-connector-c/archive/v{{ version }}.tar.gz
+  sha256: 6da2563a7940457846267989732b97028c350d7de07f89e355f15646641d0ed7
+
+build:
+  number: 0
+  missing_dso_whitelist:
+    - '*'
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - ninja
+  host:
+    - openssl
+  run:
+    - mysql
```

Comment:
I suspect you don't need the full server + libs + client here? You can search through the list of mysql components on this page: https://conda-forge.org/feedstock-outputs/index.html